go/fn/resourcelist.go

```diff
@@ -195,42 +195,68 @@ func (rl *ResourceList) UpsertObjectToItems(obj interface{}, checkExistence func
 	return nil
 }
 
+func (rl *ResourceList) LogResult(err error) {
```

Comment:
This method is not used anywhere. Even if it is used somewhere do we want to make it a public method?

Reply:
I was thinking about using it in the apply-replacements function, to log errors e.g. 

```
func (r *Replacements) Process(rl *fn.ResourceList) (bool, error) {
	if err := r.Config(rl.FunctionConfig); err != nil {
		rl.LogResult(err)
		return false, nil
	}
	...
}
```

Maybe some other function authors can find it useful too. I can remove it and define it in the apply-replacements function itself if you think it won't be useful here.

Reply:
Actually, I am now using `LogResult` in the panic recovery function in order to log the result before returning.